rcp/org.testeditor.rcp4.uatests/.classpath

```diff
@@ -1,14 +1,15 @@
-<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
now has unix line ending
(and one additional line which might not be so easy to spot:
<classpathentry kind="lib" path="lib/core-fixture-3.0.2.jar"/>)